packages/react-dom/src/__tests__/ReactDOMForm-test.js

```diff
@@ -1670,6 +1672,38 @@ describe('ReactDOMForm', () => {
     expect(divRef.current.textContent).toEqual('Current username: acdlite');
   });
 
+  it('multiple form submissions in rapid succession do not throw', async () => {
+    const formRef = React.createRef();
+    let actionCounter = 0;
+    function App() {
+      // This is a userspace action. it must take a non-zero amount of time to
+      // allow the form to be submitted again before the first one finishes.
+      // Otherwise, the form transitions will be batched and will not run sepereately.
+      async function submitForm() {
+        actionCounter++;
```

Comment:
Change to `Scheduler.log('Action')` and then below you can do `assertLog(['Action', 'Action'])`

Reply:
Fixed based on our pairing